tests/Common/Api/PublishedProduct/AbstractPublishedProductApiTestCase.php

```diff
@@ -69,31 +68,10 @@ protected function sanitizePublishedProductData(array $publishedProductData)
      */
     protected function publishProducts(array $identifiers)
     {
-        foreach ($identifiers as $identifier) {
-            $process = new Process($this->getPublishCommand($identifier));
-            $process->mustRun();
-        }
-    }
-
-    /**
-     * @param string $identifier
-     *
-     * @return string
-     */
-    private function getPublishCommand($identifier)
-    {
-        $config = $this->getConfiguration();
-        $installPath = $config['pim']['install_path'];
-        $binPath = $config['pim']['bin_path'];
-
-        $publishCommand = sprintf('%s/%s/console pim:product:publish %s', $installPath, $binPath, $identifier);
+        $commandLauncher = new CommandLauncher($this->getConfiguration());
```

Comment:
you could initialise it in the root test case in a private, and allow access to it with a function `getCommandLauncher` for example (or use a protected variable, less encapsulation).